test/test_functional_filtering.py

```diff
@@ -136,6 +136,26 @@ def test_highpass(self):
         # TBD - this fails at the 1e-4 level, debug why
         assert torch.allclose(sox_output_waveform, output_waveform, atol=1e-3)
 
+    def test_equalizer(self):
+        """
+        Test biquad peaking equalizer filter, compare to SoX implementation
+        """
+
+        CENTER_FREQ = 1000
+        Q = 0.707
+        GAIN = 1
+
+        noise_filepath = os.path.join(self.test_dirpath, "assets", "whitenoise.mp3")
```

Comment:
@vincentqb  instead of calling into sox while testing, should we write out reference files instead? especially since the asset is fixed.

Reply:
Yes, I agree with you.

Reply:
Although this could be done in a separate PR for all the other tests too.